ast.py

```diff
@@ -44,6 +44,30 @@ def build_parser():
     parser.add_argument("-parseable-output", action="store_true")
     return parser
 
+def ast_dump_file(arguments):
+    for arg in arguments:
+        if arg.startswith("-DAST_DUMP_FILE="):
+            return arg.split("=")[1]
+    return None     
+
+def dump_to_file(filename, command):
+    if filename is None:
+        return False
+
+    print("Using file %s to dump output" % filename)
+    sys.stdout.flush()
+
+    try:
+        with open(filename, "w") as outfile:
+            subprocess.call(command, stdout = outfile, stderr = outfile)  
+    except Exception as e:
+        print(e)
+        return False
+    finally:
+        sys.stdout.flush()
```

Comment:
What's this case for? Only the print of the exception? If so can you move it up there and remove the finally?

Reply:
@keith Updated